helios-client/src/main/java/com/spotify/helios/common/descriptors/Job.java

```diff
@@ -246,6 +250,10 @@ public String getHostname() {
     return hostname;
   }
 
+  public Date getCreated() {
+    return created;
+  }
```

Comment:
if Job is meant to be immutable, then this method breaks that, as Date can be altered